document/js-api/index.bs

```diff
@@ -984,3 +984,31 @@ same class of exception is thrown as for out of memory conditions in JavaScript.
 The particular exception here is implementation-defined in both cases.
 
 Note: ECMAScript doesn't specify any sort of behavior on out-of-memory conditions; implementations have been observed to throw OOMError and to crash. Either is valid here.
+
+<h2 id="limits">Implementation-defined Limits</h2>
+
+The WebAssembly core specification allows an implementation to define limits on the syntactic structure of the module.
+The standard WebAssembly JavaScript Interface described in this document hereby defines a set of standard limits on the syntactic module structure as below.
+Modules that exceed these limits are invalid and are rejected with a {{CompileError}}.
```

Comment:
I think the idea here is that there actually shouldn't be lower implementation limits, that these are exact. Should the text be clarified?

Reply:
Probably, since low-resource devices (like embedded) and computationally-sensitive uses (think: Etherum's eWASM) won't be able to cope with these limits particularly well.

Reply:
These limits only apply to JavaScript embeddings and are not relevant to others, including blockchain platforms.

Reply:
@rossberg What about JS embeddings without sufficient RAM? For one concrete example, mobile devices might have performance problems processing large WASM bundles due to a slower clock speed + less RAM available. (The max amount of allocatable memory per this, for example, is 2GB. Low-end smartphones don't usually have this much available - some only *have* 2GB of RAM.)

Reply:
PTAL at new text